### ISSUE TITLE
AOP incompatible app export (5.1)

### ISSUE THREAD

**Issue by tschf** [OraOpenSource/OXAR#238]
I think the latest version of AOP is incompatible with versions < 5.1 as I get this error. Well, probably it's just be because the app export is from 5.1.

Propped up because I ran an installation with 5.0 (technically, the docs say we support 4 and 5 releases)

```
--application/set_environment
APPLICATION 232 - AOP Sample Application (APEX 5.1)
--application/delete_application
begin
*
ERROR at line 1:
ORA-20001: API operation includes calls that are incompatible with the current
runtime.
ORA-06512: at "APEX_050000.WWV_FLOW_API", line 1323
ORA-06512: at "APEX_050000.WWV_FLOW_API", line 1379
ORA-06512: at "APEX_050000.WWV_FLOW_API", line 2101
ORA-06512: at line 2
```

**Comment by tschf** [comment]
https://github.com/OraOpenSource/OXAR/blob/master/addons/aop/aop.sh#L32

Could change this to `_50` to ensure it installs on both. Is there a difference in the two apps? Could look at the apex file name and install the appropriate app export if there is a diff.

**Comment by dgielis** [comment]
Hi Trent, yes there's a difference in both apps - but we can decide to install the 5.0 version always in OXAR. If they want the 5.1 version they could also download it from our site? Or is there an easy why we can install one or the other depending the version of APEX they choose?

AOP and APEX 4.x will technically work, but we don't support that anymore.

**Comment by tschf** [comment]
I think that's that best approach - install the 5.0 version for OXAR :+1:

**Comment by tschf** [comment]
Should be resolved from commit b2e2b47f4eef720f4f9512c0de934573d4a5a68e
